src/commands/createRepo/RepoNameStep.ts

```diff
@@ -19,7 +19,7 @@ export class RepoNameStep extends AzureWizardPromptStep<IStaticWebAppWizardConte
             undefined;
 
         wizardContext.newRepoName = (await ext.ui.showInputBox({
-            prompt: localize('AppServicePlanPrompt', 'Enter the name of the new GitHub repository. Special characters will be replaced with "-" upon creation.'),
+            prompt: localize('AppServicePlanPrompt', 'Enter the name of the new GitHub repository. We automatically build and deploy to Azure from your repo using GitHub Actions!'),
```

Comment:
How about:

> Azure Static Web Apps will automatically build and deploy using GitHub Actions.

- exclamation point seems a bit much IMO
- "We" is a lot more informal than the example screenshot you gave. Also who is "We"? Because the VS Code extension doesn't do that...  I like mentioning ASWA because we've talked about how this prompt is confusing in the sense of "How does this relate to static web apps?"
- So many "to \<this\>", "from \<this\>", "using \<this\>" feels a bit awkward. I think "to Azure" and "from your repo" can be inferred whereas "using GitHub Actions" is actually informative